Transportation/BikeHire/doc/spec.md

```diff
@@ -0,0 +1,193 @@
+Bike Hire Docking Station
+=========================
+
+Description
+-----------
+
+A bike hire docking station where subscribed users can hire and return a bike.
+It provides data about its main features and availability of bikes and slots.
+
+ 
+
+Data Model
+----------
+
+-   `id` : Unique identifier.
+
+-   `type` : Entity type. It must be equal to `BikeHireDockingStation`.
+
+-   `dateCreated` : Entity's creation timestamp
+
+    -   Attribute type: [DateTime](https://schema.org/DateTime)
+
+    -   Optional
+
+-   `dateModified` : Last update timestamp of this entity
+
+    -   Attribute type: [DateTime](https://schema.org/DateTime)
+
+    -   Optional
+
+-   `location` : Geolocation of the station represented by a GeoJSON
+    (Multi)Polygon or Point.
+
+    -   Attribute type: `geo:json`.
+
+    -   Normative References: <https://tools.ietf.org/html/rfc7946>
+
+    -   Mandatory if `address` is not defined.
+
+-   `address` : Registered docking station site civic address.
+
+    -   Normative References: <https://schema.org/address>
+
+    -   Mandatory if `location` is not defined.
+
+-   `name` : Name given to the docking station
+
+    -   Normative References: <https://schema.org/name>
+
+    -   Mandatory
+
+-   `description` : Description about the bike hire docking station.
+
+    -   Normative References: <https://schema.org/description>
+
+    -   Optional
+
+-   `image` : A URL containing a photo of this docking station.
+
+    -   Normative References: <https://schema.org/image>
+
+    -   Optional
+
+-   `totalSlotNumber` : The total number of slots offered by this bike docking
+    station.
+
+    -   Attribute type: [Number](http://schema.org/Number)
+
+    -   Allowed values: Any positive integer number or 0.
+
+    -   Optional
+
+-   `freeSlotNumber` : The number of slots available for returning and parking
+    bikes.
+
+    -   Attribute type: [Number](http://schema.org/Number)
+
+    -   Allowed values: A positive integer number, including 0. It must lower or
+        equal than `totalSpotNumber`.
+
+    -   Metadata:
+
+        -   `timestamp` : Timestamp of the last attribute update
+
+        -   Type: [DateTime](https://schema.org/DateTime)
+
+    -   Optional
+
+-   `outofserviceSlotNumber` : The number of slots that are out of order and
```

Comment:
I don't know if it has been used as such but it looks better to me as "outOfServiceSlotNumber" :) Feel free to ignore this suggestion too ;)